src/vs/workbench/contrib/files/browser/files.contribution.ts

```diff
@@ -362,6 +362,11 @@ configurationRegistry.registerConfiguration({
 			'description': nls.localize('fileDialogDefaultPath', "Default path for file dialogs, overriding user's home path. Only used in the absence of a context-specific path, such as most recently opened file or folder."),
 			'scope': ConfigurationScope.MACHINE
 		},
+		'files.dialog.currentDirectory': {
```

Comment:
I find the setting name and description not easy to understand for what it ends up doing.

Reply:
I understand that the setting name and description might not be very clear. Feel free to change them to something that better reflects what the setting actually does.